src/o365/spo/commands/site/site-set.ts

```diff
@@ -422,6 +495,13 @@ class SpoSiteSetCommand extends SpoCommand {
         }
       }
 
+      if (args.options.sharingCapabilities) {
+        let sharingCapability: SharingCapabilities = SharingCapabilities[(args.options.sharingCapabilities.trim() as keyof typeof SharingCapabilities)];        
```

Comment:
Since we have the ability to map enum to array of string, wouldn't it be easier to validate the specified value using `this.sharingCapabilities.indexOf(args.options.sharingCapability) < 0`?